spec/abstractops.html

```diff
@@ -857,6 +857,50 @@ <h1>ISO 8601 grammar</h1>
           MinuteSecond
           `60`
 
+      TimeHourNotValidMonth : one of
+          `00` `13` `14` `15` `16` `17` `18` `19` `20` `21` `23`
+
+      TimeHourNotThirtyOneDayMonth : one of
+          `02` `04` `06` `09` `11`
```

Comment:
Can we find a way to make this work that _doesn't_ require such bizarre one-off productions, e.g. early errors?